scania-digital-design-system/sdds
# fix(data table): standardise order of controls in Storybook

**Describe pull-request**  
Standardised order of controls in Storybook for data table component. Also updated descriptions, refactored deprecated controls and introduced noMinWidth controls with custom width input to all sub-stories.

**Solving issue**  
Fixes: [AB#DTS-708](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?selectedIssue=DTS-708)

**How to test**  
_Add description how to test if possible_
1. Go to Storybook link below
2. Check in Data Table -> All sub stories
3. Inspect order of controls
4. Read control descriptions

**Suggested test steps**
